docs/get-started/checkpoint-start.md

```diff
@@ -6,16 +6,19 @@ sidebar_position: 4
 
 # Start Teku from a recent state
 
-To get Teku up and running in only a few minutes, you can start Teku from a recent finalized checkpoint state rather than syncing from genesis.
+To get Teku up and running in only a few minutes, you can start Teku from a recent finalized
```

Comment:
maybe we change `you can` for `you should` to emphasise `checkpoint-sync-url` is better?